ee/tabby-ui/app/(dashboard)/components/sidebar.tsx

```diff
@@ -50,9 +60,24 @@ export default function Sidebar({ children, className }: SidebarProps) {
                   <SidebarButton href="/cluster">
                     <IconNetwork /> Cluster Information
                   </SidebarButton>
-                  <SidebarButton href="/team">
-                    <IconUsers /> Team Management
-                  </SidebarButton>
+                  <SidebarCollapsible
+                    title={
+                      <>
+                        <IconGear />
+                        Settings
+                      </>
+                    }
+                  >
+                    <SidebarButton href="/settings/general">
```

Comment:
Are 'settings/general' and '/settings/team' reserved entry points? They will result in a 404 error.